pretty.js

```diff
@@ -46,6 +46,13 @@ function filter (value) {
   return result
 }
 
+function isPinoLine (line) {
+  var requiredProperties = [ 'hostname', 'pid' ]
+  return Object.keys(line).filter(function (k) {
+    return requiredProperties.indexOf(k) > -1
+  }).length > 0
+}
```

Comment:
I think we can remove the Object.keys and loop, and just use a couple of conditions with &&. we should also check that v is 0.